docs/erp_bereitstellen.adoc

```diff
@@ -1230,6 +1230,8 @@ NOTE: Für den Versicherten wird eine Kopie des Bundles im JSON-Format inkl. ser
 |Code   |Type Success
 |200  | OK +
 [small]#Die Anfrage wurde erfolgreich bearbeitet und das Ergebnis der Anfrage wird in der Antwort übertragen.#
+|252  | Warning +
+[small]#Die Anfrage hat eine ungültige Arztnummer (LANR oder ZANR)#
```

Comment:
Der Warning Text ist laut A_24033
"Ungültige Arztnummer (LANR oder ZANR): Die übergebene Arztnummer entspricht nicht den Prüfziffer-Validierungsregeln."

Hinweis: Es ist vorgesehen, dass zu einem späteren Zeitpunkt, die fehlerhafte Prüfziffernvalidierung zu einem Abbruch anstatt einem Warning führt.

Reply:
Bitte auch in https://github.com/gematik/api-erp/blob/master/docs/erp_statuscodes.adoc ergänzen.

Reply:
Bitte wieder reviewen